config/opensearch_dashboards.yml

```diff
@@ -1,15 +1,12 @@
-# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
 #
-# Licensed under the Apache License, Version 2.0 (the "License").
-# You may not use this file except in compliance with the License.
-# A copy of the License is located at
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
 #
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# or in the "license" file accompanying this file. This file is distributed
-# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
-# express or implied. See the License for the specific language governing
-# permissions and limitations under the License.
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
 
```

Comment:
As mentioned here https://github.com/opensearch-project/opensearch-build/issues/152#issuecomment-966657660. Files like this should only have 
```
# Copyright OpenSearch Contributors
# SPDX-License-Identifier: Apache-2.0
```
Can someone check here if the header is correct? Just wanted to confirm once

Reply:
@bbarani @peterzhuamazon can you verify. Looks like the second one is for `For files that contain attribution to an external project`

Reply:
Informed Miki to verify once as well

Reply:
Lets follow this rule,
First Remove the amazon header

1. - if there is no header left, use the 2-liner
2. - If there are other headers, use the long one

Reply:
Do we need to have 
```
# Licensed under the Apache License, Version 2.0 (the "License").
# You may not use this file except in compliance with the License.
# A copy of the License is located at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# or in the "license" file accompanying this file. This file is distributed
# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
# express or implied. See the License for the specific language governing
# permissions and limitations under the License.
``` 
or it can be change to 
```
# Copyright OpenSearch Contributors
# SPDX-License-Identifier: Apache-2.0
#
# The OpenSearch Contributors require contributions made to
# this file be licensed under the Apache-2.0 license or a
# compatible open source license.
#
# Modifications Copyright OpenSearch Contributors. See
# GitHub history for details.
```

Reply:
This is the one that I use and confirmed by Open Source Team:
```

# SPDX-License-Identifier: Apache-2.0
#
# The OpenSearch Contributors require contributions made to
# this file be licensed under the Apache-2.0 license or a
# compatible open source license.

```

Reply:
Looks like all the plugins teams have followed
```
# Copyright OpenSearch Contributors
# SPDX-License-Identifier: Apache-2.0
```
I would suggest let's have the same for consistency and as mentioned by Miki over here https://github.com/opensearch-project/opensearch-build/issues/152#issuecomment-966657660